server/model/promo.js

```diff
@@ -0,0 +1,11 @@
+import {Record} from 'immutable';
+
+export const Promo = Record({
+  modifiers: null,
+  url: null,
+  image: null,
+  meta: null,
+  title: null,
+  copy: null,
+  type: null
+});
```

Comment:
Could we have something like:
`Promo(article: Article, modifiers: List[String])`?

That way we have one representation of the article that we can use everywhere?